Test/git-issues/git-issue-3187a.dfy

```diff
@@ -0,0 +1,10 @@
+// RUN: %exits-with 2 %baredafny verify %args "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
+
+module P {
+  const c = 5
+  method m() {
+    var x = 5;
+    var z := ( var y = 5; y);
```

Comment:
The parentheses and final `;` are not required syntax. Is this test meant to test these variations or can they just as well be omitted here?

Reply:
I jus thought the parentheses made it clearer. The test here is just that the = sign (which I'll admit is an error I can easily make, given 20 years of Java, and 20 more of C and C++ and Fortran) is wrong.

But the semicolons are needed -- lines 6 and 7 are statements/declarations in a method body. Right?

Reply:
Oh, my bad, I was reading the local variable declarations as field declaration. So, yes, the semi-colons are needed.